src/Npgsql/PostgresMinimalDatabaseInfo.cs

```diff
@@ -26,7 +26,7 @@ class PostgresMinimalDatabaseInfo : PostgresDatabaseInfo
         static readonly PostgresBaseType[] Types = typeof(NpgsqlDbType).GetFields()
             .Select(f => f.GetCustomAttribute<BuiltInPostgresType>())
             .Where(a => a != null)
-            .Select(a => new PostgresBaseType("pg_catalog", a.Name, a.OID))
+            .Select(a => new PostgresBaseType("pg_catalog", a!.Name, a!.OID))
```

Comment:
They didn't take care of recognizing `Where(a => a != null)`? In any case it seems cleaner to do a `.Cast<BuiltInPostgresType>()` than to use bang...